Pronto-MIA/Pronto-MIA/DataAccess/ProntoMiaDbContext.cs

```diff
@@ -45,7 +45,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
            modelBuilder.ApplyConfiguration(new DeploymentPlanTypeConfig());
            modelBuilder.ApplyConfiguration(new FcmTokenTypeConfig());
 
-           var generatorOptions = new Pbkdf2GeneratorOptions(
+           /*var generatorOptions = new Pbkdf2GeneratorOptions(
```

Comment:
Is this commented code needed anymore?

Reply:
Theoretically no but it is the code that generated the user `Franz` within the first migration so once we make the application production ready we will most probably need this code again.